src/plone/restapi/interfaces.py

```diff
@@ -163,3 +163,15 @@ def parse(idx_query):
         have been reconstructed to the proper data types that the adapted
         index expects.
         """
+
+
+class IBlockTransformer(Interface):
+    """ A multiadapter that can post-process (transform) the value of a block
+    """
+
+    def transform(value, event):
```

Comment:
Instead of having the event passed here as an argument, would you rather have the event being provided in the adapter lookup, for a three way adapter?

```
queryMultiAdapter(
            [obj, request, event], IBlockTransformer, name=block.get('@type', ''))
```

Reply:
Seems fine for me!